frontend/src/features/landing_page/components/GeoMap.tsx

```diff
@@ -47,4 +50,29 @@ export const GeoMap = () => {
       {markers}
     </MapContainer>
   );
+
+  function handleClick() {
+    setIsVisible(true);
+  }
+
+  let content;
+
+  if (isVisible) {
+    content = map;
+  } else {
+    content = (
+      <div className="text-center" id="placeholder">
+        <img
+          src="/permaplant-map-placeholder.png"
+          alt="PermaplanT Map"
+          className="mb-2 hover:cursor-pointer"
+          onClick={handleClick}
+        />
+        <i>{t('geomap:hint')}</i>
+        <br />
+      </div>
+    );
+  }
+
+  return <>{content}</>;
```

Comment:
```suggestion
  return content;
```

We don't need the fragment here.